.github/workflows/test-cluster-set-up.yml

```diff
@@ -32,7 +32,7 @@ jobs:
             echo $distribution_type $security
             stackName=ODFE-$distribution_type-SECURITY-$security
 
-            existingStacks=`aws cloudformation list-stacks --stack-status-filter CREATE_COMPLETE --query 'StackSummaries[*].StackName' --output text`
```

Comment:
Why remove this?

Reply:
Because it was filtering only the created ones and not the rollback or updated ones. Now it will list all stacks irrespective of the status